src/aws/cloud-formation-wrapper.ts

```diff
@@ -97,10 +97,19 @@ class CloudFormationWrapper {
             {},
         );
 
-        // filter stacks by given stackName
+        // filter stacks by given stackName and check by nested stack RootId
         const filteredStackNames = stacks
-            .map((stack) => stack.StackName) // collect list of stack names
-            .filter((name) => name.includes(stackName)); // filter by stackName
+          .reduce((acc, stack) => {
+              if (!stack.RootId) {
+                  return acc;
+              }
```

Comment:
@matteobattista Hey, thank you for your PR!

How we are going to handle cases without `serverless-aws-nested-stacks`?
The current logic will give us an empty list as any of the items will not have the `RootId`

Reply:
Hi, this code is inside the function:

Reply:
yeah, I have already forgot about this logic )

Reply:
Will the logic below work for you?
```
//filter stacks by given stackName and check by nested stack RootId
const filteredStackNames = stacks
     .filter((stack) => stack.RootId && stack.StackName.includes(stackName)) // filter by stackName
     .map((stack) => stack.StackName); // collect list of stack names
```

Reply:
Unfortunately this check  `&& stack.StackName.includes(stackName))` is too weak for us because a parent `stackName` can start like others, for example:
`my-project-name`
`my-project-name-sls`

With the regEx check:
```
const regex = new RegExp(`\/${stackName}\/`);
const match = stack.RootId.match(regex);
```
we should be certain that the nested stack is a child of the right father.

Reply:
Do you need to check it in the `RootId` value or could it be a `stack.StackName`?

Reply:
I'm just curious why are you checking matching in the `RootId`

Reply:
The first check, filters the nested stacks, the second (by regEx) to understand if it belongs to the right parent... did I answer?

Reply:
Yes, thank you!
Could you please resolve the conflict?

Reply:
Looks good, thank you!